tests/integration/test_private_results.py

```diff
@@ -0,0 +1,38 @@
+from ddt import ddt, unpack, data
+from selenium.common.exceptions import NoSuchElementException
+from base_test import PollBaseTest
+
+
+@ddt
+class TestPrivateResults(PollBaseTest):
+    """
+    Check the functionality of private results.
+    """
+    @unpack
+    @data(('Survey Private', ['enjoy', 'recommend', 'learn']), ('Poll Private', ['choice']))
+    def test_private_results(self, page_name, names):
```

Comment:
@Kelketek I'm slightly against such a long test methods, as they are usually testing for more than one thing. This one tests for:

* You can clink on every input, and in the end feedback would not be shown
* There's a submit button with "Submit" text after that
* After submitting once, submit button text changes to "Resubmit"
* ... and feedback is displayed.
* ... and there's no `poll-top-choice` in html of some element
* ... and poll-footnote is not on the page
* ... and there are `form` on the page.

Most of this line items could be a separate test. Since this is an integration test it might be ok to do all this checks in one test to save test initialization time. So, up to you, but I would suggest splitting this test into several smaller, for the benefit of better test scenario isolation and clearer "What was broken" messages when something break.